### PR TITLE
[SELS-TASK] Student: Create markup for Dashboard Page

### PR DESCRIPTION
**Pull request by balagtasangelo**
[Commands]
            None.
[Pre-conditions]
            1. Register an account.
            2. Login with an existing user account.
[Expected Output]
            1. Show Dashboard Page for student.
            2. Show Dashboard Page for student.
[Notes]
            None.

### CODE REVIEW

File: resources/views/__dashboard.blade.php [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    </div>
+
+    <div class="flex-1 list-group p-3">
+            <p>
+                {{ Auth::user()->name }}
+            </p>
+            <a class="underline" href="#">Learned 20 Words</a>
+            <a class="underline" href="#">Learned 5 Lessons</a>
+    </div>
+</div>
```
Comment (jeremiahC) [inline resources/views/__dashboard.blade.php]: if ka bantay ka aning naay murag no entry sign, meaning always dapat naa kay space at the end of the file.

File: webpack.mix.js [hunk 2]
```diff
@@ -11,6 +11,8 @@ const mix = require('laravel-mix');
...
  */
 
-mix.js('resources/js/app.js', 'public/js')
-    .react()
-    .sass('resources/sass/app.scss', 'public/css');
+mix.js('resources/js/app.js', 'public/js');
+
+mix.postCss("resources/css/app.css", "public/css", [
+    require("tailwindcss")
+]);
```
Comment (jeremiahC) [inline webpack.mix.js]: add space next line

File: resources/css/main.css [hunk 3]
```diff
@@ -0,0 +1,4 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
```
Comment (jeremiahC) [inline resources/css/main.css]: remove extra space

**Comment by jeremiahC** [comment]
* can you change the background sa home page to white dili ma klaro ang mga text
* [Image from Gyazo](https://gyazo.com/1389fe671ec30179ec36a1f08805028d)

File: app/View/Components/Categories.php [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+    /**
+     * Get the view / contents that represent the component.
+     *
+     * @return \Illuminate\Contracts\View\View|string
+     */
+    public function render()
+    {
+        return view('components.categories');
+    }
```
Comment (jeremiahC) [inline app/View/Components/Categories.php:27]: if wala ni gamit na file delete nalang

File: resources/views/layouts/app.blade.php [hunk 5]
```diff
@@ -18,66 +18,81 @@
...
+                                    </div>
+                                </li>
+                            @endguest
+                        </ul>
+                    </div>
                 </div>
-            </div>
-        </nav>
+            </nav>
+        </header>
```
Comment (jeremiahC) [inline resources/views/layouts/app.blade.php]: create a separate partial file for navbar para dali ra nato ma debug.

* create a folder under sa `views` and name it `layouts` and inside it please create file `navbar.blade.php`
After that you can just `@include` directive sa laravel para ma append sa `app.blade.php`

File: routes/web.php [hunk 6]
```diff
@@ -2,21 +2,12 @@
...
-|
-*/
-
 Route::get('/', function () {
     return view('welcome');
 });
 
 Auth::routes();
 
 Route::get('/home', [App\Http\Controllers\HomeController::class, 'index'])->name('home');
```
Comment (jeremiahC) [inline routes/web.php]: I suggest ipangan ni og Dashboard nalang, ato lang ibase ang naming sa specification para dali masabtan. and Sa files related ani kay Dashboard nato iname.

Controller => DashboardController.php
View =>  dashboard.blade.php


and then under dashboard view we will create partial files.
   => activities.blade.php
   => user-details.blade.php



Purpose of this kay mas organize sya tanawn, mas dali i debug, mas limpyo. We can discuss this tomorrow

File: resources/views/home.blade.php [hunk 7]
```diff
@@ -1,23 +1,25 @@
...
+                    Activities
+                </div>
+
+                <hr class="mx-5">
+
+                <div class="p-3">
+                    Activities Content
                 </div>
             </div>
         </div>
```
Comment (jeremiahC) [inline resources/views/home.blade.php]: We can create a separate file for this one and name it as `activities.blade.php`,  and we will append in this file using `@include` directive

File: resources/views/__dashboard.blade.php (hunk 1 above)
Comment (balagtasangelo) [reply resources/views/__dashboard.blade.php]: I'll check the PR every time mo create ko new sir, kay di nako ma kita ang no entry sign sa VScode, and will commit the changes. (bow)

**Comment by balagtasangelo** [comment]
>     * can you change the background sa home page to white dili ma klaro ang mga text
> 
>     * [Image from Gyazo](https://gyazo.com/1389fe671ec30179ec36a1f08805028d)

I couldn't recreate it on my end, but seeing from the code, I think it triggers dark mode. Changing the code to no dark mode.

File: app/View/Components/Categories.php (hunk 4 above)
Comment (balagtasangelo) [reply app/View/Components/Categories.php:27]: Updated the changes under the Categories Page PR.  

link: https://github.com/framgia/ph_sels_laravel_Angelo/pull/5/commits/eea21bdf39f9253c579177158dfb8cf2e69b6c1d

File: routes/web.php (hunk 6 above)
Comment (balagtasangelo) [reply routes/web.php]: Done

File: resources/views/home.blade.php (hunk 7 above)
Comment (balagtasangelo) [reply resources/views/home.blade.php]: Done

File: resources/views/layouts/app.blade.php (hunk 5 above)
Comment (balagtasangelo) [reply resources/views/layouts/app.blade.php]: Done

File: webpack.mix.js (hunk 2 above)
Comment (balagtasangelo) [reply webpack.mix.js]: Done

File: resources/css/main.css (hunk 3 above)
Comment (balagtasangelo) [reply resources/css/main.css]: Done

**Review by jeremiahC — APPROVED** [review]

**Pull request merged**